### PR TITLE
[Glance] adding image_admin and image_viewer roles

### PR DESCRIPTION
**Pull request by rajivmucheli**
Hi @Carthaca,

I was able to run tempest pipeline for all services in qa, application credentials worked fine as well. Please validate.

Regards,
Rajiv

### CODE REVIEW

File: openstack/glance/templates/seeds.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ spec:
 
   roles:
   - name: cloud_image_admin
+  - name: image_admin
```
Comment (Carthaca) [inline openstack/glance/templates/seeds.yaml]: image_viewer is missing

File: openstack/glance/templates/etc/_glance-policy.json.tpl [hunk 2]
```diff
@@ -1,50 +1,67 @@
 {
     "context_is_cloud_admin":  "role:cloud_image_admin",
     "context_is_admin":  "rule:context_is_cloud_admin",
+    "owner": "project_id:%(project_id)s",
+    "admin": "role:image_admin and rule:owner",
+    "member": "role:member and rule:owner",
+    "viewer": "role:image_viewer and rule:owner",
+    "context_is_image_admin":  "rule:context_is_admin or role:admin or rule:admin",
+    "context_is_image_viewer":  "rule:context_is_image_admin or rule:viewer or role:member",
```
Comment (Carthaca) [inline openstack/glance/templates/etc/_glance-policy.json.tpl]: role:member -> r**u**le: member

File: openstack/glance/templates/etc/_glance-policy.json.tpl [hunk 3]
```diff
@@ -1,50 +1,67 @@
 {
     "context_is_cloud_admin":  "role:cloud_image_admin",
     "context_is_admin":  "rule:context_is_cloud_admin",
+    "owner": "project_id:%(project_id)s",
+    "admin": "role:image_admin and rule:owner",
+    "member": "role:member and rule:owner",
+    "viewer": "role:image_viewer and rule:owner",
+    "context_is_image_admin":  "rule:context_is_admin or role:admin or rule:admin",
```
Comment (Carthaca) [inline openstack/glance/templates/etc/_glance-policy.json.tpl]: remove role:admin

File: openstack/glance/templates/etc/_glance-policy.json.tpl [hunk 4]
```diff
@@ -1,50 +1,67 @@
...
-    "get_member": "",
-    "get_members": "",
-    "modify_member": "",
+
+    "add_image": "rule:context_is_image_admin",
+    "delete_image": "rule:context_is_image_admin",
+    "get_image": "rule:context_is_image_viewer",
+    "get_images": "rule:context_is_image_admin",
+    "modify_image": "rule:context_is_image_admin",
+    "publicize_image": "rule:context_is_image_admin",
```
Comment (Carthaca) [inline openstack/glance/templates/etc/_glance-policy.json.tpl]: only cloud admins should be able to do this, change back to `rule:context_is_cloud_admin` please

**Comment by rajivmucheli** [comment]
working on https://github.com/sapcc/helm-charts/pull/1328

**Pull request closed without merging**